mjpc/tasks/tasks.cc

```diff
@@ -34,6 +35,12 @@ namespace {
 // sized reference afterward. This way the compiler enforces equality between
 // kNumTasks and the size of the array initializer.
 const TaskDefinition<const char*> kTasksArray[]{
+    {
```

Comment:
@hartikainen can you add this example after stand and walk since these changes will now be on the main branch?

Reply:
Fixed.